.travis.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+language: node_js
+
+node_js:
+  - 10/*
+
+sudo: required
+
+services:
+  - docker
+
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - libstdc++-5-dev
```

Comment:
Shouldn't the packages listed in the README be added here?
E.g.

```
granite
gtk+-3.0
glib-2.0
jpegoptim
optipng
```

Reply:
The tool automatically pulls the dependencies from your project.